force-app/main/default/quickActions/Review__c.SubmitReview.quickAction-meta.xml

```diff
@@ -0,0 +1,53 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<QuickAction xmlns="http://soap.sforce.com/2006/04/metadata">
+    <description>Submits this Review</description>
```

Comment:
@nicolersmith This is the proposed new quick action to perform the 'Submit Review' action. It is a lightweight quick action, but will need a label, description, and the success message. We can go over this first if that helps! I will also update the WI with a demo gif.